config/broker/README.md

```diff
@@ -1,4 +1,8 @@
-# NATS Streaming - simple installation
+# NATS - simple installation
+
+## 1. NATS Streaming Deprecation Notice :warning:
+> The NATS Streaming Server is being deprecated.  Critical bug fixes and security fixes will be applied until June of 2023.  NATS enabled applications requiring persistence should use [JetStream](https://docs.nats.io/jetstream/jetstream).
```

Comment:
Do you really want to commit to almost 2 years of support here?

Reply:
No, we will just support several releases，2023 is the official promise of NATS STREAMING